app/scripts/modules/kubernetes/src/pipeline/stages/runJob/configureJob.html

```diff
@@ -0,0 +1,115 @@
+<form name="form" class="container-fluid form-horizontal" style="padding: 0;">
+  <div>
+    <v2-modal-wizard heading="Configure Run Job"
+                     dismiss="ctrl.cancel">
+      <v2-wizard-page label="Basic Settings" mark-complete-on-view="false">
+        <stage-config-field label="Account">
+          <account-select-field component="ctrl.stage"
+                                field="account"
+                                provider="'kubernetes'"
+                                accounts="ctrl.accounts"
+                                on-change="ctrl.updateNamespaces()">
+          </account-select-field>
+        </stage-config-field>
+      
+        <stage-config-field label="Namespace" help-key="kubernetes.namespace">
+          <namespace-select-field component="ctrl.stage"
+                                  field="namespace"
+                                  hide-label="true"
+                                  columns="12"
+                                  account="ctrl.stage.account"
+                                  namespaces="ctrl.namespaces">
+          </namespace-select-field>
+        </stage-config-field>
+    
+        <stage-config-field label="Image">
+          <ui-select name="containerSelect" ng-model="ctrl.stage.container" class="form-control input-sm" required>
+              <ui-select-match>{{ ctrl.stage.container.imageDescription | kubernetesImageId }}</ui-select-match>
+              <ui-select-choices
+                repeat="container in ctrl.containers | orderBy: 'imageDescription.imageId'"
+                group-by="ctrl.groupByRegistry"
+                refresh-delay="0"
+                ui-disable-choice="container.message"
+                refresh="ctrl.searchImages($select.search)">
+                <span ng-bind-html="container.imageDescription | kubernetesImageId | highlight: $select.search"></span>
+              </ui-select-choices>
+            </ui-select>
+        </stage-config-field>
+
+        <stage-config-field label="Container Name">
+            <div>
+              <input type="text" class="form-control input-sm" ng-model="ctrl.stage.container.name">
+            </div>
+          </stage-config-field>
+      </v2-wizard-page>
+      <v2-wizard-page label="Container" done="true">
```

Comment:
From your screenshots, the contents of this section look too far to the left. Could you please center them somehow? It might be nothing more than wrapping the children of this `v2-wizard-page` in
```
<div class="form-group">
  ...
</div>
```

Reply:
@danielpeach yeah, they are a bit inconsistent. with the labels on the left (stage layout), they look really "smooshed". having the wizard on the left takes up too much of the modal real estate for it to look good. perhaps we should take about solutions on Slack?

Reply:
Example:
![image](https://user-images.githubusercontent.com/3324110/32735557-103f9380-c863-11e7-8e60-051398c78ee0.png)